hack/common.sh

```diff
@@ -252,7 +252,17 @@ os::build::build_static_binaries() {
 #     then just the host architecture is built.
 os::build::build_binaries() {
   # Create a sub-shell so that we don't pollute the outer environment
-  (
+  ( os::build::internal::build_binaries "$@" )  
+}
+
+# Build binaries targets specified. Should always be run in a sub-shell so we don't leak GOBIN
```

Comment:
Seems like "binaries targets" here and on line 261 should be "binary targets" instead.